src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs

```diff
@@ -760,6 +888,19 @@ internal virtual void GetCandidateExtensionMethods(
         {
         }
 
+#nullable enable
+        /// <summary>
+        /// Return the extension containers from this specific binding scope
+        /// Since the lookup of extension members is iterative, proceeding one binding scope at a time,
+        /// GetExtensionContainers should not defer to the next binding scope. Instead, the caller is
+        /// responsible for walking the nested binding scopes from innermost to outermost. This method is overridden
+        /// to search the available members list in binding types that represent types, namespaces, and usings.
+        /// </summary>
+        internal virtual void GetExtensionContainers(ArrayBuilder<NamedTypeSymbol> extensions, Binder originalBinder)
```

Comment:
Should the terminology here be "extension declaration" instead of "extension container"?